src/vue-components/inflections-table-prerendered.vue

```diff
@@ -0,0 +1,115 @@
+<template>
```

Comment:
We use it to display inflection tables for Greek paradigms, where inflection tables itself are stored within JSON files. I think this can be merged with a component that renders "regular" tables, but this will require some effort: paradigm tables has some particularities that we do not support in "regular" views, such as supplemental tables, table links etc. 

I think it would be good to merge the two to simplify the workflow, but that will require some time for code changes and especially testing. Not sure what would be the best time for this. What do you think?

Reply:
ah, okay, I see. No, I don't think merging them is a high priority right now.  Let's wait until after the release and we've had a chance to begin some of the other bigger refactoring that's needed. Thanks!